packages/manager/test/lib.test.ts

```diff
@@ -6,20 +6,56 @@ import Ceramic from '@ceramicnetwork/http-client'
 import { DIDDataStore } from '@glazed/did-datastore'
 import { model as idxModel } from '../src/__generated__/model'
 import { idxModelManager } from './utils'
+import type { TileDocument } from '@ceramicnetwork/stream-tile'
 
 import { Manager } from '../src'
 
 describe('3ID Manager', () => {
   jest.setTimeout(60000)
 
-  const ceramic = new Ceramic('http://localhost:7777')
+  const ceramic = new Ceramic('http://localhost:7777', { syncInterval: 30 * 60 * 1000 })
   const dataStore = new DIDDataStore({ ceramic, model: idxModel })
 
   beforeAll(async () => {
     const manager = idxModelManager(ceramic)
     await manager.toPublished()
   })
 
+  test('tile loader cache fail example', async () => {
```

Comment:
@PaulLeCam pushed up an example of the issue here, probably will need some help on it, been pretty stuck on it
logged output in comments, and disabling cached results in expected outputs, as mentioned before have not been able to replicate in a more simple example in the tileloader tests

Reply:
👍 thanks for the setup, I'll dig into it